Automattic/woocommerce-payments
# Fix platform checkout Store API nonce check

#### Changes proposed in this Pull Request

This PR removes the code that disables the store API nonce check for the platform checkout.

#### Testing instructions

- Checkout on `fix/store-api-authentication` in the platform checkout repository
- Make sure you can perform a transaction using the platform checkout

-------------------


**Post merge**
